tests/helpers/airflow_tests/test_airflow_wrapper.py

```diff
@@ -917,3 +917,48 @@ def dag_parallel():
     dag_def = dag_parallel()
     assert len(tasks_list) == 1
     dag_def.test()
+
+
+def callable_source():
+    @dlt.resource
+    def test_res():
+        context = get_current_context()
+        yield [
+            {"id": 1, "tomorrow": context["tomorrow_ds"]},
+            {"id": 2, "tomorrow": context["tomorrow_ds"]},
+            {"id": 3, "tomorrow": context["tomorrow_ds"]},
```

Comment:
Add a context field just to be able to check it (as this thing is actually executed outside of the main loop)